Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:latest
+
+RUN sudo apt-get install -y software-properties-common && sudo add-apt-repository ppa:virtuslab/git-machete && sudo apt-get update && sudo apt-get install -y python3-git-machete 
```

Comment:
Okay! so as first thing... pls split this `RUN` into a few lines so that commenting on individual lines is easier!

Reply:
Having it in a single line actually reduces the number of layers which makes the image altogether a lot smaller and efficient.

Reply:
Oh! you mean a single **Dockerfile instruction** (`RUN` in this case), yes... but for the sake of readability, the instruction can be split with `\`, as done e.g. in https://github.com/VirtusLab/git-machete/blob/master/ci/deb-ppa-upload/Dockerfile#L11